### ISSUE TITLE
calico instructions

### ISSUE THREAD

**Issue by npsonis** [pontussk/calico#4]
Hello,
Can you provide details on how to use your software? The README file does not provide any info.

The only thing that I have found online is coming from Palkopoulou et al 2018, but it is just descriptive:

Contamination rates were then estimated using SAMtools v.0.1.19 with the ‘mpileup’ option and the extended BAQ computation (-E) and the python script ‘calico.0.2.py’ (https://github.com/pontussk/calico). 

Thanks in advance!

**Comment by npsonis** [comment]
Ok I typed python2 calico.0.2.py --help and also checked the python script and found my answer.

I added the default values.

I guess as input files it needs the ref genome (haploid e.g. mitogenome) in fasta and the BAM file with the mapped reads to the same reference. 

It needs as prerequisite the SAMtools software (in the PATH).

Usage: calico.0.2.py [options] arg1 arg2

Options:
  -h, --help                                              show this help message and exit
  --notransitions                                     notransitions (default=False)
  --nodamage                                         nodamage (default=False)
  --mindepth=MINDEPTH                      mindepth (default=3)
  --maxdepth=MAXDEPTH                     maxdepth (default=1000)
  --verbose                                             verbose  (default=False)
  --noheader                                          noheader (default=False)
  --indels                                                indels   (default=True)
  --noindels                                             noindels   (default=False)
  --samtoolspath=SAMTOOLSPATH       samtoolspath  (default='samtools')
  -m MAPQ, --requiremapq=MAPQ       mapq   (default=0)
  -q QUAL, --requirebasequal=QUAL     qual   (default=0)
  --outputsites                                        outputsites  (default=False) 
  -r REF_FILE, --ref_file=REF_FILE            ref_file     (default=False)
  -b BAM, --bam=BAM                           bam          (default=False)
  -o OUTFILE, --outfile=OUTFILE             outfile      (default=False)
  --distribution                                        distribution (default=False)

Unfortunately i cannot make it run. I am using rCRS.fa (from NCBI) and a BAM file (human mapped reads to rCRS), I typed python2 calico.0.2.py PATHTOMYBAM/reads.bam PATHTOMYREF/rCRS.fa and it does not run, I tried using some options, but again it did not printed anything, it just stack without doing anything (I checked with top and nothing is running in the background).

I also tried the old 0.1.9 version of samtools (I even re-mapped my reads using bwa and this version of samtools) but again the issue remains.

May I have some help with this issue?

Thanks in advance

**Comment by npsonis** [comment]
I think I found the solution to run it...

samtools-0.1.19/samtools mpileup -E sample.bam | python2 calico.0.2.py -o calico.txt --outputsites (other options can be used)

But now the output informs me that 0 sites were used. 

SITES 	MAJ 	MIN 	CONT 	CI_low 	CI_up 	 Informative_sites

0 	0 	0 	NA 	NA	NA	NA

I tried it with mindepth of 1 but it returns the same.

**Comment by pontussk** [comment]
Yes that is correct. Thanks for your interest, we are working towards
publishing this tool.

Best,
Pontus

**Comment by npsonis** [comment]
Hi pontussk,

So, the 0 sites is normal?

All the best,

Nikos

**Comment by npsonis** [comment]
Hi pontussk,
can you please let me know if the output is what I should expect. It seems that it didn't work...
Best,
Nikos

**Comment by arenvale** [comment]
Hi @npsonis! 
Were you able to run the program successfully? I'm having the same problem as you, and I can't find detailed instructions on how Calico is supposed to run.
Thanks!
Valeria

**Comment by npsonis** [comment]
Hi @arenvale, this was 5-6 years ago, but I think I abandoned it after getting what I mention above (0 sites).

**Comment by arenvale** [comment]
Great, thank you very much for your reply!
